docs/using/examples/environment.rst

```diff
@@ -16,7 +16,7 @@ Create the Virtual Machines
 You can reuse the Virtual Machines defined in the Vagrant configuration for :ref:`the MongoDB tutorial <tutmongo>`.
 If you have since shutdown or destroyed those VMs, boot them up again:
 
-.. code-block:: console
+.. prompt:: bash alice@mercury:~/flocker-tutorial$ auto
 
    alice@mercury:~/flocker-tutorial$ vagrant up
    Bringing machine 'node1' up with 'virtualbox' provider...
```

Comment:
Sphinx prompt creates HTML where the prompt is not copied. You can specify what a non-copyable prompt is.

```
.. prompt:: bash foo$

   xxx
   yyy
```

renders:

```
foo$ xxx
foo$ yyy
```

But if you want the two to have different prompts, you use the `auto` keyword and then use the specified prompt(s) in the written code. Then those prompts aren't copied.